docs/sdkx-cocos2dx/deep-linking.mdx

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 120
+title: Deep Linking in FAQs
+description: "Using custom URL schemes to support deep linking in your app."
+---
+
+import {
+  Admonition,
+  TabItem,
+  Image,
+  Intro,
+} from "@site/src/components/forDocs";
+
+# Deep Linking in FAQs {#android-deep-links}
```

Comment:
The link shouldnt contain "android" i guess

Reply:
Will fix this